app/src/main/res/values/strings.xml

```diff
@@ -20,14 +20,14 @@
     <string name="mode_animation">Animation</string>
     <string name="mode_laser">Laser</string>
     <string name="send_button">SEND</string>
-    <string name="preview_button">PREVIEW</string>
     <string name="permission_required">Permission required</string>
-    <string name="txt_turn_on_bluetooth">Turn on Bluetooth and try again</string>
     <string name="no_device_found">No device found. Please try again.</string>
     <string name="select_drawable_text">Select Drawable:</string>
-    <string name="select_drawable">Please select Drawable to be previewed</string>
     <string name="character_not_found">Some of the character(s) entered cannot be displayed.</string>
     <string name="send_text">Send Text</string>
     <string name="send_drawables">Send Drawables</string>
     <string name="invertLED">Invert LED</string>
+    <string name="radio_error">Select option before proceeding</string>
```

Comment:
Where is this used?

Reply:
Ah I removed the use. Thanks sir, just a min

Reply:
Have loik

Reply:
Fixed, have a look sir